new_jersey/ver2/style.css

```diff
@@ -0,0 +1,171 @@
+    font-weight: normal;
+    font-style: normal;
+}
+
+.container {
+    background-color: #4e5860;
+    width: 800px;
+    height: 600px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.widget {
+    background-color: #f1f8f9;
+    width: 388.99px;
+    height: 464px;
+    border-radius: 10px 10px 29px 29px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    box-shadow: 0px 37px 76px 2px rgba(0, 0, 0, 0.2);
+}
+.header {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 16px;
+    padding-right: 17px;
+    border-radius: 10px 10px 0 0;
+}
+.main {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-left: 5px;
+    padding-top: 10px;
+}
+
+.forecast {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 30px;
+}
+
+.info {
+    display: flex;
+    align-items: flex-start;
+    font-family: "Proxima Nova Light";
+    font-size: 58px;
+    margin-left: 20px;
+}
+.temperature {
+    color: #a5acaf;
+}
+.info img {
+    padding-left: 5px;
+    padding-top: 16px;
+}
+
+.description_location {
+    display: flex;
+    flex-direction: column;
+    margin-right: 37px;
+}
+.description {
+    font-family: "SourceSansPro-Light";
+    color: #bac2c6;
+    font-size: 24px;
+}
+.location {
+    font-family: "SourceSansPro-Semibold";
+    color: #a2a9ad;
+    font-size: 16px;
+}
+
+.date {
+    background-color: #54bae6;
+    width: 65px;
+    height: 69px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: "Open Sans", Sans-Serif;
+    font-weight: 700;
+    color: #ffffff;
+}
+.month {
+    padding-top: 5px;
+    text-transform: uppercase;
+}
+
+.footer {
+    background-color: #2e3335;
+    width: 388.99px;
+    height: 136px;
+    border-radius: 0px 0px 10px 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.additional-info {
+    width: 388.99px;
+    margin-top: 46px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-family: "Proxima Nova Bold";
+    color: #ffffff;
+    font-size: 14px;
+    text-transform: uppercase;
+}
+.wind {
+    margin-left: 40px;
+    width: 90px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+}
+.rain {
+    width: 62px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin-left: -10px;
+}
+.sun {
+    margin-right: 53px;
+    width: 67px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+}
+
+.scroll-dots {
```

Comment:
I would suggest adding `display: flex;` to this block so that all 3 dots are displayed at the same height.